cfme/tests/configure/test_visual_cloud.py

```diff
@@ -1,18 +1,16 @@
 # -*- coding: utf-8 -*-
 import pytest
 
-
 from cfme import test_requirements
 from cfme.configure.settings import visual
-from cfme.cloud.availability_zone import AvailabilityZone
-from cfme.cloud.provider import CloudProvider
-from cfme.cloud.flavor import Flavor
+from cfme.cloud.availability_zone import AvailabilityZone, AvailabilityZoneView
```

Comment:
**Required:** here should be AvailabilityZoneAllView as AvailabilityZoneView matches only header

Reply:
@rlbabyuk I have made the changes